Subject: Add Ctrl+Alt+Shift+Esc key command for logind's SecureAttentionKey
From: n3rdopolis bluescreen_avenger@verizon.net Mon May 5 22:29:18 2025 -0400
Date: Wed Jun 11 09:32:34 2025 +0200:
Git: 4b89c39eea1cb89dda597d81831fc385db3f8cbc

logind now supports a new key binding https://github.com/systemd/systemd/pull/29542
Ctrl+Alt+Shift+Esc that emits SecureAttentionKey to allow login managers to start
or switch back to the greeter

--- a/virtManager/details/console.py
+++ b/virtManager/details/console.py
@@ -119,6 +119,7 @@ def build_keycombo_menu(on_send_key_fn):
 
     make_item("<Control><Alt>BackSpace", ["Control_L", "Alt_L", "BackSpace"])
     make_item("<Control><Alt>Delete", ["Control_L", "Alt_L", "Delete"])
+    make_item("<Control><Alt><Shift>Escape", ["Control_L", "Alt_L", "Shift_L", "Escape"])
     menu.add(Gtk.SeparatorMenuItem())
 
     for i in range(1, 13):
